core/src/services/hdfs/backend.rs

```diff
@@ -68,6 +70,24 @@ impl HdfsBuilder {
 
         self
     }
+
+    /// Set kerberos_ticket_cache_path of this backend
+    ///
+    /// This should be configured when kerberos is enabled.
+    pub fn kerberos_ticket_cache_path(&mut self, kerberos_ticket_cache_path: &str) -> &mut self {
```

Comment:
We can return `&mut Self` here.

Reply:
Yes. I found the compile command is `cargo build --features services-hdfs` and then could fix these failure